Sources/Stream-Feed-UIKit-IOS/Views/Collection View Cells/VideoCell.swift

```diff
@@ -6,119 +6,28 @@
 //
 
 import UIKit
-import AVKit
 import Reusable
 
 public final class VideoCell: UICollectionViewCell, NibReusable {
     
     @IBOutlet weak var showControlsButton: UIButton!
     @IBOutlet weak var playButtonView: UIView!
-    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
-    @IBOutlet weak var videoView: UIView!
+    @IBOutlet weak var videoThumbnailImageView: UIImageView!
     @IBOutlet public weak var removeButton: UIButton!
-    var streamingURL: String?
     var logErrorAction: ((String, String) -> Void)?
-    private(set) var videoPlayerWrapper: VideoPlayerWrapperProtocol = VideoPlayerWrapper()
-    var showControlsState: ((Bool) -> Void)?
-    var videoPlayerTapped: (() -> Void)?
     
     public override func awakeFromNib() {
         super.awakeFromNib()
         // Initialization code
-        
-        videoPlayerWrapper.logErrorAction = logErrorAction
-        activityIndicator.removeFromSuperview()
-        activityIndicator.stopAnimating()
     }
     
     public override func prepareForReuse() {
         super.prepareForReuse()
-        self.videoPlayerWrapper.playerController?.showsPlaybackControls = false
-        self.showControlsState?(self.videoPlayerWrapper.playerController?.showsPlaybackControls ?? false)
-        self.videoPlayerWrapper.player?.pause()
-        self.videoPlayerWrapper.playerController?.player?.seek(to: .zero) { [weak self] _ in
-            guard let self = self else { return }
-            self.pauseVideo()
-        }
         removeButton.removeTap()
+        videoThumbnailImageView.image = nil
     }
     
-    func presentIndicator() {
-        activityIndicator.color = .white
+    func config(thumbnailImage: UIImage?) {
+        videoThumbnailImageView.image = thumbnailImage
     }
-    
-    
-    
```

Comment:
Why do we delete this section ? @SherifShokry123

Reply:
@Jimmy-Camelan  While implementing the timeline to support the video feature, I transferred the logic of the video cell from the camlist to the SDK repo. However, in the GetStream, the 'Video Cell' now exclusively displays the video thumbnail image without requiring any action to play the video.
There is no need to add it to the Video Wrapper; I just need to display the thumbnail image. 
Users can remove it by tapping on the close button. Therefore, this logic is not utilized.
We can revert it when needed 

![Simulator Screenshot - iPhone 15 Pro - 2023-12-31 at 17 52 07](https://github.com/camelan/swift-activity-feed/assets/107892568/e1973aa7-a129-4f6d-9649-fc92d75c3cf8)